cocos/gfx/webgl/webgl-commands.ts

```diff
@@ -22,114 +22,119 @@
  THE SOFTWARE.
 */
 
-import { debugID, error, errorID, CachedArray, cclegacy, assertID } from '../../core';
-import { WebGLCommandAllocator } from './webgl-command-allocator';
 import { WebGLEXT } from './webgl-define';
 import { WebGLDevice } from './webgl-device';
 import {
     IWebGLGPUInputAssembler, IWebGLGPUUniform, IWebGLAttrib, IWebGLGPUDescriptorSet, IWebGLGPUBuffer, IWebGLGPUFramebuffer, IWebGLGPUInput,
     IWebGLGPUPipelineState, IWebGLGPUShader, IWebGLGPUTexture, IWebGLGPUUniformBlock, IWebGLGPUUniformSamplerTexture, IWebGLGPURenderPass,
 } from './webgl-gpu-objects';
 import {
-    BufferUsageBit, ClearFlagBit, ClearFlags, ColorMask, CullMode, Format, BufferTextureCopy, Color, Rect,
+    BufferUsageBit, ColorMask, CullMode, Format, BufferTextureCopy, Color, Rect,
     FormatInfos, FormatSize, LoadOp, MemoryUsageBit, ShaderStageFlagBit, UniformSamplerTexture,
     TextureFlagBit, TextureType, Type, FormatInfo, DynamicStateFlagBit, BufferSource, DrawInfo,
     IndirectBuffer, DynamicStates, Extent, getTypedArrayConstructor, formatAlignment, Offset, alignTo,
     TextureBlit, Filter,
 } from '../base/define';
-import { WebGLStateCache } from './webgl-state-cache';
+
+import { WebGLConstants } from '../gl-constants';
+import { assertID, debugID, error, errorID } from '../../core/platform/debug';
+import { cclegacy } from '../../core/global-exports';
+
+function max (a: number, b: number): number {
+    return Math.max(a, b);
+}
```

Comment:
It adds one function invoking, i don't think it is not a good idea to do it like this.

Reply:
UPDATED:

Use function reference instead of defining a function for Math.ceil and Math.max
```ts
const max = Math.max;
```

Reply:
https://github.com/cocos/cocos-engine/pull/17529/commits/5035f0b59c007c7385865e589d0a13b59f6fc3fb